components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1892,6 +1897,11 @@ void AdsServiceImpl::ConfirmAd(
     const ads::AdInfo& info,
     const ads::ConfirmationType confirmation_type) {
   rewards_service_->ConfirmAd(info.ToJson(), confirmation_type);
+
+  if (confirmation_type.value() == ads::ConfirmationType::kViewed) {
+    brave_ads::RecordEventInWeeklyStorage(profile_->GetPrefs(),
+        "brave.weekly_storage.p2a_ad_view_confirmation_count");
```

Comment:
I think `kAdViewConfirmationCountPrefName` should be used here.

Reply:
@simonhong `kAdViewConfirmationCountPrefName` is defined in the `ads_p2a.cc` implementation, but we could create an analogous variable here and use that instead of string

Reply:
How about creating `components/brave_ads/common/pref_names.h` and define `kAdViewConfirmationCountPrefName` in that file? It's common pattern for pref names.

Reply:
Oh, we alread have it `brave/components/brave_ads/common/pref_names.h`.

Reply:
fixed!